doc/NextGen_ON_CONUS.md

```diff
@@ -161,14 +160,85 @@ For an example taking into account more realistic contributions, you can try:
 
+# Resource Usage
+
+The following table lists the CPU wall clock ime used for various realization configurations running 10 day simulation time. The timing values reported in the table are from single run, not from average. Note in particular that the `Initialization Time` may be significantly affected by system loads at the time of job start.
+
+| Realization | Number of CPUs | Initialization Time (s) | Computation Time (s) | Total Time (s) |
+| ------------- | :-----: | :--------: | :--------: | :--------: |
+| conus_bmi_multi_realization_config_w_sloth.json | 32 | 2618.6 | 737.4 | 3356.0 |
+| conus_bmi_multi_realization_config_w_sloth_noah.json | 32 | 1360.1 | 2143.9 | 3504.0 |
+| conus_bmi_multi_realization_config_w_sloth_noah_pet.json | 32 | 3204.0 | 2106.5 | 5310.5 |
+| conus_bmi_multi_realization_config_w_sloth_noah_pet_cfe.json | 32 | 1214.9 | 4069.2 | 5284.1 |
+| conus_bmi_multi_realization_config_w_sloth_noah_pet_smp.json | 32 | 1453.4 | 3087.0 | 4540.4 |
+| conus_bmi_multi_realization_config_w_sloth_noah_pet_smp_sft.json | 32 | 3245.7 | 3808.1 | 7053.8 |
+| conus_bmi_multi_realization_config_w_sloth_noah_pet_smp_sft_cfe.json | 32 | 1354.7 | 5283.1 | 6637.8 |
+
+The abreviation used for submodule names in the table:
+* noah - noah-owp-modular
+* pet - potential evapotranspiration
+* cfe - Conceptual functional equivalence
+* smp - SoilMoistureProfiles
+* sft - SoilFreezeThaw 
+
 # Run Computation with Topmodel
 
 To be added
 
 # Run Computation with Routing
 
-To be added
+To run computation on CONUS with routing, we need to build the executable with routing option turned on. This can be done using the build script displayed in the [Build the Executable](#build-the-executable) and set `-DNGEN_WITH_ROUTING:BOOL=ON`. Make sure that Python is also on, and you can run the script to build the `ngen` executable.
+
+You also need to build the `t-route` submodule. First, as the automatically downloaded `t-route` is out of date, to get the latest version, you need to remove the old `t-route` and run `git clone https://github.com/NOAA-OWP/t-route` in the `extern` directory. For building `t-route` in general, we refer user to the documentation [PYTHON_ROUTING.md](PYTHON_ROUTING.md) for essential details. We just want to add some additional discussion here to make the process easier. Note that there are more than one ways to build the `t-route`. Do `cd t-route`, then run:
+
+```
+FC=mpif90 NETCDFINC=<path-to-netcdf-include-directory> ./compiler.sh
+```
+
+This will let you to use the NetCDF library that you want to build `t-route`. Alternatively, you can edit the `compiler.sh` file, note these comment lines in the script:
```

Comment:
```suggestion
This will let you use the NetCDF library that you want to build `t-route`. Alternatively, you can edit the `compiler.sh` file, note these comment lines in the script:
```